### DISCUSSION TITLE
Variable that contains all current track IDs

### DISCUSSION THREAD

**Discussion by rsorbello** [mikel-brostrom/yolo_tracking#735] [Q&A] [answered]
Hi I am working on a program that utilizes this repo, and was hoping to see if there is a quick answer to my question before digging too much into the code.  I am trying to basically build a dictionary that gives a specific label to all the current track Ids.  In order to avoid this dictionary getting too large with longer videos and many new objects coming into the frame, I would like to just to grab all the current track ids.  I see that the 'outputs' variable contains this information, but would this also include track ids for objects that are currently being occluded? If not, is there a list that pops out the track id once the object has been occluded or been out of the frame for too long?

**Comment by mikel-brostrom** [comment, marked as answer]
You can access all the active tracks from

https://github.com/mikel-brostrom/yolov8_tracking/blob/847c517e14de861fcc9ac8faa3152aa408999a54/trackers/strongsort/sort/tracker.py#L50

Even the occluded ones, at least for the time they are alive

**Reply by rsorbello** [reply]
Thank you!
